src/pal/src/config.h.in

```diff
@@ -135,6 +135,7 @@
 #cmakedefine01 HAS_FTRUNCATE_LENGTH_ISSUE
 #cmakedefine01 UNWIND_CONTEXT_IS_UCONTEXT_T
 #cmakedefine01 HAVE_FULLY_FEATURED_PTHREAD_MUTEXES
```

Comment:
is `HAVE_FULLY_FEATURED_PTHREAD_MUTEXES` used anymore after this change? If not, can you please delete it everywhere?